app/containers/ITunesContainer/AllTracksContainer/index.js

```diff
@@ -0,0 +1,190 @@
+/**
+ *
+ * AllTracksContainer
+ *
+ */
+
+import React, { useState, useEffect, memo } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { debounce, isEmpty, get } from 'lodash';
+import styled from 'styled-components';
+import { injectSaga } from 'redux-injectors';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { Link } from 'react-router-dom';
+import { Card, Input, Skeleton } from 'antd';
+import If from '@app/components/If';
+import T from '@app/components/T';
+import For from '@app/components/For';
+import ITunesCard from '@app/components/ITunesCard';
```

Comment:
nit: `@components/T` we don't need to `@app/components/...`

Reply:
Will make the change